src/main/java/clients/TestResultsClient.java

```diff
@@ -265,12 +266,12 @@ public Response callPostTestResults(Object object) {
     public Response callGetTestResults(String vin) {
 
         Response response = given()
-//                .log().all()
                 .filters(new BasePathFilter())
                 .contentType(ContentType.JSON)
                 .pathParam("vin", vin)
+                .log().all()
                 .get("/test-results/{vin}");
-
+        System.out.println("results: " + response.then().log().all());
```

Comment:
Is this a duplicate of the log().all() on line 272?

Reply:
fixed